### PR TITLE
sys-kernel/raspberrypi-image: Add live ebuild

### PR DESCRIPTION
**Pull request by ralimi**
Package-Manager: Portage-2.3.6, Repoman-2.3.1

### CODE REVIEW

**Comment by ralimi** [comment]
Any feedback on this?

**Comment by ralimi** [comment]
+ @xmw specifically to draw attention

File: sys-kernel/raspberrypi-image/raspberrypi-image-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	insinto /lib/modules
+	doins -r modules/*
+	insinto /boot
+	newins boot/kernel.img kernel.img
+	newins boot/kernel7.img kernel7.img
+
+	readme.gentoo_create_doc
+}
+
+DOC_CONTENTS="Please configure your ram setup by editing /boot/config.txt"
```
Comment (monsieurp) [inline sys-kernel/raspberrypi-image/raspberrypi-image-9999.ebuild]: Could you define this variable just below `EGIT_REPO_URI`?

File: sys-kernel/raspberrypi-image/raspberrypi-image-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+DESCRIPTION="Raspberry PI boot loader and firmware"
+HOMEPAGE="https://github.com/raspberrypi/firmware"
+SRC_URI=""
+
+LICENSE="GPL-2 raspberrypi-videocore-bin"
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+DEPEND=""
```
Comment (monsieurp) [inline sys-kernel/raspberrypi-image/raspberrypi-image-9999.ebuild]: Please remove `DEPEND` and `RDEPEND`. They are not used anyway.

File: sys-kernel/raspberrypi-image/raspberrypi-image-9999.ebuild (hunk 1 above)
Comment (ralimi) [reply sys-kernel/raspberrypi-image/raspberrypi-image-9999.ebuild]: Done.

File: sys-kernel/raspberrypi-image/raspberrypi-image-9999.ebuild (hunk 2 above)
Comment (ralimi) [reply sys-kernel/raspberrypi-image/raspberrypi-image-9999.ebuild]: Done.

**Pull request closed without merging**